pkg/action/uninstall.go

```diff
@@ -198,7 +199,12 @@ func (u *Uninstall) deleteRelease(rel *release.Release) (string, []error) {
 
 	resources, err := u.cfg.KubeClient.Build(strings.NewReader(builder.String()), false)
 	if err != nil {
-		return "", []error{errors.Wrap(err, "unable to build kubernetes objects for delete")}
+		remainingErr := k8serrors.FilterOut(err, func(err error) bool {
```

Comment:
https://github.com/helm/helm/blob/25aa9d0e659679a3e3c2cda303b886c1f5148a20/pkg/action/upgrade.go#L249-L260

Maybe we should keep the same implementation as `upgrade` dose.

Reply:
I have to disagree. The goal of the upgrade is that resources should be created/upgraded and if that cannot be achieved, the process should fail.

In the uninstall case the goal is that resources should be gone, and if certain resources are already gone - because their API types are gone - then the goal is achieved, and those errors should not result in the process to fail. 

This is why I used the `FilterOut` helper to remove only these types of errors, bot not others.

Reply:
Yes, I mean the error returned by `u.cfg.KubeClient.Build` to process.

Reply:
@liuming-dev could you please clarify how the patch should be changed exactly?

Reply:
No action necessary here.